azblob/list.go

```diff
@@ -92,12 +108,11 @@ func (azp *Storer) FilteredList(ctx context.Context, tagsFilter string, opts ...
 		Items:      resp.Blobs,
 	}
 
-	r.Marker = resp.NextMarker
```

Comment:
Why did you remove this line ? I think this completely breaks paging doesn't it ?

Reply:
its superfluous - r.Marker already has that value